src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,48 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+
+    @Test
+    public void split(){
+        String[] data = "1,2".split(",");
+        assertThat(data).contains("1");
+
+        assertThat(data).containsExactly("1", "2");
```

Comment:
```suggestion
        assertThat(data).containsExactly("1", "2");
```

`containsExactly`에서 "1"이 있음을 검증했으므로 위의 `contains`는 제거되어도 좋겠네요 :)